src/containers/MoviePage.jsx

```diff
@@ -5,110 +5,34 @@ import { connect } from 'react-redux';
 
 import { Paper } from '@material-ui/core';
 
-import { PAGE_COUNT, PARAMS as params } from '../constants/constants';
+import { PAGE_COUNT } from '../constants/constants';
 import SearchContainer from './SearchContainer';
 import FlatPagination from '../components/FlatPagination';
-import MovieList from '../components/MovieList';
+import MovieListContainer from './MovieListContainer';
 import {
-  request,
-  succsess,
-  failure,
   reset,
+  setPagination,
```

Comment:
I don't think that name setPaginaion is good for this case.
To avoid name duplication you can use aliases when importing. For example setPage as setPageAction.